test/unit/test_serialize_debug.c

```diff
@@ -67,8 +67,8 @@ bool test_debug_serialize_save() {
 
 	rz_core_file_close(core->file);
 	free(core);
```

Comment:
rz_core_free

Reply:
That causes crashes (on both x86 and ARM64).

Reply:
Try doing rz_core_free after sdb_free i.e. as the last free call

Reply:
Still fails with the same error.

Reply:
Fixed the error. It was because `rz_core_free` called `rz_bp_free`, which was tryIng to free stack allocated strings. There should be API functions for changing the name and other member values of  `RzBreakpoint`. Right now, it is done bare, without any API (see the handler for "dbn": https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_debug.c#L3456-L3472), which makes it easier to make mistakes like `bp->name = "breakp"`, which then lead to memory bugs.

Reply:
Shall I add such API functions?

Reply:
Yes, please do